Pod/Classes/Cards/Card.swift

```diff
@@ -13,15 +13,32 @@ import Foundation
 */
 public struct Card {
 
+    /// The card number that is shown on the user's card.
     public let bankCardNumber: Number
+    
+    /**
+     The card verification code for the user's card.
+     
+     - important: You should never save and/or autofill a user's card verification code in your app, as this will void the CVC's purpose as a security feature.
+     - important: CVCs that are validated by *Caishen* as `CardValidationResult.Valid` are not necessarily validated by the card issuer as well. In order to validate the CVC, you should opt for the integration of a [Payment Gateway](https://en.wikipedia.org/wiki/Payment_gateway) to validate the authenticity of the entered CVC.
```

Comment:
Thanks for pointing this out. However, in this case, I would argue that this is more related to the meaning of "Payment Gateway" and not directly related to the definition of a CVC and it might be confusing to have this located in a `-seealso` block.